File: examples/url/shorten/go/shortenALongUrl.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/url"
)

// Shortens a destination URL and returns a full short URL.
func ShortenAlongUrl() {
	urlService := url.NewUrlService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := urlService.Shorten(&url.ShortenRequest{})
	fmt.Println(rsp, err)
}

File: examples/js/url/shorten/shortenALongUrl.js
const { UrlService } = require("m3o/url");

// Shortens a destination URL and returns a full short URL.
async function shortenAlongUrl() {
  let urlService = new UrlService(process.env.MICRO_API_TOKEN);
  let rsp = await urlService.shorten({
    destinationURL: "https://mysite.com/this-is-a-rather-long-web-address",
  });
  console.log(rsp);
}

shortenAlongUrl();
